rm-datasource/src/main/java/io/seata/rm/datasource/sql/druid/BaseRecognizer.java

```diff
@@ -50,6 +57,20 @@ public BaseRecognizer(String originalSQL) {
 
     }
 
+    public void excuteVisit(SQLExpr where, SQLASTVisitor visitor) {
```

Comment:
Please change the name to executeVisit()

Reply:
fixed